views/partials/jira-configuration-table.hbs

```diff
@@ -78,6 +78,17 @@
                 >
                   {{connection.syncStatus}}
                 </span>
+              {{#if ../isIncrementalBackfillEnabled}}
+                {{#if (isAllSyncSuccess connection)}}
+                  <span class="jiraConfiguration__info__backfillDate">
+                      {{#if connection.backfillSince}}
+                        Backfilled till: <span class="jiraConfiguration__info__backfillDate-label" data-backfill-since="{{ toISOString connection.backfillSince }}">{{ connection.backfillSince }}</span>
```

Comment:
Can we change the wording to `Backfilled until` instead?

Reply:
Updated to `from`.